CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
+tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+- Started keeping a changelog!
+
+## [0.44.3] - 2025-01-17
+
+### Added
+
+- Add conformance tests for MPolyRing, and fix bugs in ^ and is_unit for MPoly [PR
+  #1950](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1950)
+
+- Add ignore_kwargs to @attr macro [PR
```

Comment:
This will get deprecated again in https://github.com/Nemocas/AbstractAlgebra.jl/pull/1966, so no real need to add it at all.

Reply:
I would disagree, to the contrary, someone may have stumbled over the new feature and used it, and then it is suddenly gone. So instead of just never mentioning it, if the changelog has an entry where it gets added and then another where it gets deprecated, that helps users to adjust.

(It may not be very important here because it was so short lived but I don't see why to deviate from the principle here)

But this should be formatted with `code blocks`:
```suggestion
- Add optional `ignore_kwargs` argument to `@attr` macro [PR
```